shortfin/README.md

```diff
@@ -1,20 +1,52 @@
 # shortfin - SHARK inference library and serving engine
 
-## Simple User Installation
+The shortfin project is SHARK's open source, high performance inference library
+and serving engine. Shortfin consists of these major components:
 
-Install:
+* The "libshortfin" inference library written in C/C++ and built on
+  [IREE](https://github.com/iree-org/iree)
+* Python bindings for the underlying inference library
+* Example applications in
+  ['shortfin_apps'](https://github.com/nod-ai/SHARK-Platform/tree/main/shortfin/python/shortfin_apps)
+  built using the python bindings
 
+## Prerequisites
+
+* Python 3.11+
+
+## Simple user installation
+
+Install the latest stable version:
+
+```bash
+python -m pip install shortfin
```

Comment:
```suggestion
pip install shortfin
```
We may want to harmonize this with the root README.

Reply:
Sure. In some cases I like to be explicit with `python -m`, but user docs are likely worth trimming to the shorter form.